it/uk/gov/hmrc/emcstfe/support/IntegrationBaseSpec.scala

```diff
@@ -60,11 +61,13 @@ trait IntegrationBaseSpec
     startWireMock()
     DownstreamStub.onSuccess(DownstreamStub.POST, "/write/audit", OK, Json.obj())
     DownstreamStub.onSuccess(DownstreamStub.POST, "/write/audit/merged", OK, Json.obj())
+    sys.props -= SendToEIS.configName
   }
 
   override protected def afterAll(): Unit = {
-    stopWireMock()
     super.afterAll()
+    sys.props -= SendToEIS.configName
```

Comment:
It makes sure after each test that the `SendToEIS` feature switch is not present when other integration tests run. If this line wasn't present then there could be failures in other tests because the `SendToEIS` property is present

Reply:
(I encountered that issue when I was running the tests for this PR)